src/SourceGenerators/Uno.UI.SourceGenerators.Internal/DependencyObject/DependencyPropertyGenerator.cs

```diff
@@ -315,6 +332,34 @@ private void GenerateSource(SourceProductionContext context, ImmutableArray<Gene
 			{
 				using (builder.BlockInvariant($"partial class {dpCandidatesData[0].ContainingTypeName}"))
 				{
+					var completeEnumsCount = boolPropertiesCount / 8;
+					for (int i = 0; i < completeEnumsCount; i++)
+					{
+						builder.AppendLineIndented($"private GeneratedDependencyPropertyFlags{i} _generatedDependencyPropertyFlags{i};");
+						using (builder.BlockInvariant($"private enum GeneratedDependencyPropertyFlags{i} : byte"))
```

Comment:
I think we can use uint32, byte will likely get padded for memory alignment.

Reply:
@jeromelaban Do you mean using uint for **all** enums? or only complete ones?

For example, if we need 35 booleans, do we generate one `uint` and one `byte`? or two `uint`s?